Text/Chapter Text/Chapter 1.txt

```diff
@@ -0,0 +1,164 @@
+We were raised as your siblings,[N]
+and I've never forgotten[AN]
+my gratitude to your family.[AN]
+Even so, I've also never forgotten[N]
+my heritage as a commoner.[AN]
+Even after my sister Nada[N]
+accepted your marriage proposal,[AN]
+truly making us nobility,[N]
+I have not forgotten my heritage.[A]
+
+[....v.]
+So?[N]
+
+[.v....]
+So I can't help but feel sorry[N]
+for these people.[AN]
+I knew the "Donateenary Tax"[N]
+was a massive tax, but I had[AN]
+no clue it was this bad.[N]
+These people won't be able to[AN]
+survive much longer under[N]
+these conditions[A]
+
+[....v.]
+I hear you, but it's not my problem.[AN]
+The tax only affects our sworn enemies,[N]
+the Donateenaries, I'm sure if they[AN]
+renounce their faith,[N]
+the tax will no longer affect them.[A]
+
+[.v....]
+Vedge...[A][X]
+
+## Chapter1_NorthVillage
+[....v.][LoadCurrent]
+[.v....][LoadAwakeningAnna]
+Oh hey![N]
+You're the Nameless Knights of Void, right?[AN]
+My sisters have been talking about them.[N]
+I had a boyfriend who went off to[AN]
+join you guys but I haven't heard[N]
+any word from him since.[AN]
+Maybe he got lost? I'll join you[N]
+guys and offer my thieving services,[AN]
+hopefully I'll run into him while I[N]
+travel with you guys.[A][X]
+
+## Chapter1_SouthVillage
+[....v.][LoadCurrent]
+[.v....][LoadFace][0x61][0x01]
+You know, many people here[N]
+support the rebels.[AN]
+The Queen's harsh taxes of late[N]
+on imports and foodstuffs have[AN]
+really made people really detest[N]
+Queen Void's policies.[AN]
+I understand that despair, people[N]
+really don't like losing the money[AN]
+they worked so hard to earn,[N]
+but I dunno, the Kingdom at large[AN]
+has its own bills to pay too,[N]
+right? Maybe Queen Void's just[AN]
+having as hard times as we are,[N]
+and she needs some money?[AN]
+Oh I'm sorry for keeping you[N]
+here, take this, it's a memento[AN]
+of my father, but I'd rather[N]
+see it used by brave knights[AN]
+like you than gather dust on[N]
+my mantle.[A][X]
+
+
+## Chapter1_House
+[....v.][LoadCurrent]
+[.v....][LoadNotArch]
+Bah. Can't your lot of knights[N]
+just leave us the hell alone?![AN]
+It's bad enough that we're paying[N]
+most of our livelihoods on your[AN]
+godsforsaken taxes, but now you're[N]
+going after Jazzack too?! Right[AN]
+when Jazzack had gotten his life[N]
+on the straight and narrow, that[AN]
+bitch of a queen raised taxes so[N]
+high he couldn't keep his armory[AN]
+open. He had to sell it to the[N]
+government to avoid going into debt,[AN]
+but any money he made from that he[N]
+immediately had to give back to[AN]
+the government to sate their tax[N]
+rates. All because he, like many[AN]
+other poor souls in this city,[N]
+descend from Donateenary immigrants[AN]
+that came here fleeing their own[N]
+country. Those people came to our[AN]
+nameless kingdom seeking freedom[N]
+and new opportunities. Instead, all[AN]
+they get is a life where violent[N]
+rebellion seems to be their only[AN]
+option...[A][2NL]
```

Comment:
~~replace `[AN]`s with `[N]`s for the full Arch rant experience~~

Reply:
I had that at first but it's actual good exposition so